### PR TITLE
Add POST to resource with two variables step

### PR DESCRIPTION
**Pull request by jurriaan**
Needed for https://github.com/blendle/core-api/pull/2570

### CODE REVIEW

**Comment by blendle-hubbit** [comment]
Based on the modified files, and historical file contributions, I am tagging @JeanMertz and @broeken to review this PR.

File: lib/cucumber/blendle/version.rb [hunk 1]
```diff
@@ -6,6 +6,6 @@ module Cucumber
   # This module defines default steps used by all of Blendle Ruby projects.
   #
   module BlendleSteps
-    VERSION = '0.9.0'.freeze
+    VERSION = '0.9.1'.freeze
```
Comment (broeken) [inline lib/cucumber/blendle/version.rb]: @JeanMertz once told me it's bad practice to update version numbers in the same PR as a new feature. Not sure if I've remembered this correctly.

File: lib/cucumber/blendle/steps/resource_steps.rb [hunk 2]
```diff
@@ -59,6 +59,20 @@
...
+# * And the client does a POST request to the "user_item_preference" resource with user_id "bartsimpson" and item_id "bnl-telegraaf-20130212-1" and the following content:
+#   """json
+#   {
+#     "prefer": "less"
+#   }
+#   """
+#
+When(/^the client does a (POST|PUT) request to the "([^"]*)" resource with (\S+) "([^"]*)" and (\S+) "([^"]*)" and the following content:$/) do |method, resource, key, value, key2, value2, content|
+  api  = HyperResource.from_body(api_body)
+  url  = api.send(resource, key => value, key2 => value2).href
```
Comment (broeken) [inline lib/cucumber/blendle/steps/resource_steps.rb:71]: `key: value, key2: value2`?

**Review by broeken — APPROVED** [review]

File: lib/cucumber/blendle/steps/resource_steps.rb (hunk 2 above)
Comment (jurriaan) [reply lib/cucumber/blendle/steps/resource_steps.rb:71]: That results in something different (is equivalent to `{ :key => value, :key2 => value2 }`, which is not what we want here)

**Review by JeanMertz — APPROVED** [review]

**Pull request merged**